plugins/spark_upgrade/tests/resources/accessing_execution_plan/input/sample.scala

```diff
@@ -0,0 +1,7 @@
+package org.piranha
+
+object AccessingExecutionPlan {
+  def main(args: Array[String]): Unit = {
+    var r = df.queryExecution.executedPlan.collect
```

Comment:
should this be `df.queryExecution.asInstanceOf[AdaptiveSparkPlanExec].executedPlan.collect`?